cmake/Platform/Windows/Packaging/PostInstallSetup.wxs

```diff
@@ -61,5 +62,11 @@
             Execute="deferred"
             Impersonate="no"/>
 
+        <CustomAction Id="UnRegisterEngine"
+            ExeCommand="&quot;[SystemFolder]cmd.exe&quot; /C @&quot;[root.scripts]o3de.bat&quot; register --remove --this-engine"
```

Comment:
This '@' character is important if we ever decide to include any quoted path arguments in the future.  Without it you get some fun quote stripping behavior